server/server/src/static_files.rs

```diff
@@ -30,6 +30,7 @@ pub fn config_static_files(cfg: &mut web::ServiceConfig) {
     cfg.service(
         web::resource("/sync_files/{table_name}/{record_id}")
             .route(web::get().to(sync_files))
+            .route(web::delete().to(delete_sync_file))
```

Comment:
Wondering if we should change both this and the `get` to use `/sync_files/{table_name}/{record_id}/file_id` instead of `/sync_files/{table_name}/{record_id}?id={file_id}` particularly for delete, it feels a bit wrong to use a query parameter rather than route path. Not that it really matters much!

Reply:
yeah, I like that idea. it's nice to use the `/file_id` pattern. agree it doesn't matter too much.. but it is nicer!